python/ray/dataframe/test/test_dataframe.py

```diff
@@ -41,7 +47,7 @@ def test_ftypes(ray_df, pandas_df):
 
 @pytest.fixture
 def test_values(ray_df, pandas_df):
-    assert(np.array_equal(ray_df.values, pandas_df.values))
+    np.testing.assert_equal(ray_df.values, pandas_df.values)
```

Comment:
When `np.nan` is there, equality always is `False`, so we use numpy's test for our asserts.